system-tests/lib/cre/contracts/contracts.go

```diff
@@ -141,6 +142,114 @@ func (d *dons) allDonCapabilities() []keystone_changeset.DonCapabilities {
 	return out
 }
 
+func (d *dons) toV2ConfigureInput(chainSelector uint64, contractAddress string) cap_reg_v2_seq.ConfigureCapabilitiesRegistryInput {
+	nops := make([]capabilities_registry_v2.CapabilitiesRegistryNodeOperator, 0)
+	nodes := make([]capabilities_registry_v2.CapabilitiesRegistryNodeParams, 0)
+	capabilities := make([]capabilities_registry_v2.CapabilitiesRegistryCapability, 0)
+	donParams := make([]capabilities_registry_v2.CapabilitiesRegistryNewDONParams, 0)
+
+	// Collect unique capabilities and NOPs
+	capabilityMap := make(map[string]capabilities_registry_v2.CapabilitiesRegistryCapability)
+	nopMap := make(map[string]capabilities_registry_v2.CapabilitiesRegistryNodeOperator)
+
+	for _, don := range d.donsOrderedByID() {
+		// Extract capabilities
+		for _, cap := range don.Capabilities {
+			capID := fmt.Sprintf("%s@%s", cap.Capability.LabelledName, cap.Capability.Version)
+			if _, exists := capabilityMap[capID]; !exists {
+				capabilityMap[capID] = capabilities_registry_v2.CapabilitiesRegistryCapability{
+					CapabilityId:          capID,
+					ConfigurationContract: common.Address{},
+					Metadata:              []byte("{}"),
```

Comment:
metadata should have two keys: `capabilityType` and `responseType` from the V1 config